src/subgroups/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _subteams:
```

Comment:
Shouldn't this be added to the main TOC in `src/index.rst`?

Reply:
I have the subgroups index in the main index, so it should be listed now, right?